src/main/webapp/jsp/instructorFeedbackEdit.jsp

```diff
@@ -1,1324 +1,58 @@
-<%@ page import="teammates.common.datatransfer.StudentAttributes" %>
-<%@ page import="teammates.common.datatransfer.InstructorAttributes" %>
-<%@ page import="teammates.logic.core.Emails.EmailType" %>
-<%@ page import="teammates.ui.controller.InstructorFeedbackEditPageData" %>
-<%
-    InstructorFeedbackEditPageData data = (InstructorFeedbackEditPageData)request.getAttribute("data");
- %>
-<!DOCTYPE html>
-<html>
-<head>
-    <title>TEAMMATES - Instructor</title>
-
-    <link rel="shortcut icon" href="/favicon.png" />
-
-    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-
-    <link type="text/css" href="/bootstrap/css/bootstrap.min.css" rel="stylesheet"/>
-    <link type="text/css" href="/bootstrap/css/bootstrap-theme.min.css" rel="stylesheet"/>
-    <link type="text/css" href="/stylesheets/teammatesCommon.css" rel="stylesheet"/>
-
-    <!--[if lt IE 9]>
-        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
-        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
-    <![endif]-->
-
-    <script type="text/javascript" src="/js/googleAnalytics.js"></script>
-    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
-    <script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/jquery-ui.min.js"></script>
-    <script type="text/javascript" src="/js/common.js"></script>
-    <script type="text/javascript" src="/bootstrap/js/bootstrap.min.js"></script>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 
-    <jsp:include page="../enableJS.jsp"></jsp:include>
-    
-    <link rel="stylesheet" href="/stylesheets/datepicker.css" type="text/css">
-
-    <script type="text/javascript" src="/js/datepicker.js"></script>
-    <script type="text/javascript" src="/js/instructor.js"></script>
-    <script type="text/javascript" src="/js/instructorFeedbacks.js"></script>
-    <script type="text/javascript" src="/js/instructorFeedbackEdit.js"></script>
-</head>
+<%@ page import="teammates.common.util.Const" %>
 
-<body onload="readyFeedbackEditPage();
+<%@ taglib tagdir="/WEB-INF/tags" prefix="t" %>
+<%@ taglib tagdir="/WEB-INF/tags/instructor" prefix="ti" %>
+<%@ taglib tagdir="/WEB-INF/tags/instructor/feedbackEdit" prefix="feedbackEdit" %>
+<%@ taglib tagdir="/WEB-INF/tags/instructor/feedbacks" prefix="feedbacks" %>
+
+<c:set var="jsIncludes">
+        <link rel="stylesheet" href="/stylesheets/datepicker.css" type="text/css" media="screen">
+        
+        <script type="text/javascript" src="/js/datepicker.js"></script>
+        <script type="text/javascript" src="/js/instructor.js"></script>
+        <script type="text/javascript" src="/js/instructorFeedbacks.js"></script>
+        <script type="text/javascript" src="/js/instructorFeedbackEdit.js"></script>
```

Comment:
Indentation is off here and at the `<c:if>` in the body.
Also, it may be best arranging the `<c:set>` for the constant into a single line.